docker/dockerfile.merlin

```diff
@@ -93,15 +93,15 @@ RUN ln -s /usr/bin/python3 /usr/bin/python
 
 RUN pip install --no-cache-dir --upgrade pip; pip install --no-cache-dir "cmake<3.25.0" ninja scikit-build pandas==1.5.2 \
                 fastrlock nvidia-pyindex pybind11 pytest \ 
-                transformers==4.12 tensorflow-metadata betterproto \
+                transformers tensorflow-metadata betterproto \
                 cachetools graphviz nvtx scipy "scikit-learn<1.2" \
                 tritonclient[all]==2.29.0 grpcio-channelz fiddle wandb npy-append-array \
```

Comment:
Hi @bashimao , please help to remove the version(and description above) for tritonclient, thanks